ext/proj4_c_impl/errors.c

```diff
@@ -0,0 +1,31 @@
+#ifndef RGEO_PROJ4_ERRORS_INCLUDED
+#define RGEO_PROJ4_ERRORS_INCLUDED
+
+#include <ruby.h>
+
+#include "preface.h"
+
+#ifdef RGEO_PROJ4_SUPPORTED
+
+#include "errors.h"
+
+RGEO_BEGIN_C
+
+VALUE error_module;
+VALUE rgeo_error;
+VALUE rgeo_invalid_projection_error;
+
+void rgeo_init_proj_errors() {
+  VALUE rgeo_module;
+
+  rgeo_module = rb_define_module("RGeo");
+  error_module = rb_define_module_under(rgeo_module, "Error");
```

Comment:
I’m a tiny bit unsure wether these addresses should be marked or not. Maybe @teckwan you have the answer to that question?

Otherwise I’ll test that next week

Reply:
> GC should know about global variables which refer to Ruby's objects, but are not exported to the Ruby world.

I'm not entirely sure but at first glance I would say that it's necessary to mark it. It isn't done in `rgeo` as well. We however didn't run to any SEGVs due to this unmarked object, but it surely must be due to it not being referred to often. @BuonOmo @keithdoggett

Reply:
Okay I’ll try and reproduce the bug then. It seems similar to https://github.com/msgpack/msgpack-ruby/issues/133

Reply:
I tried to create a segfault by modifying these modules, but was unable to do so. I can still use `rb_gc_register_address` on them to mark them though if there's no downside.